notebooks/rbc-omnisci-udtf-normalize.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>Maybe replace the last comment by: <code>return the size of output column</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xnd-project/rbc/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>